### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by kRkk**
Reference to #1

### CODE REVIEW

File: i18n/pl-pl.ts [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+    </message>
+    <message>
+        <location filename="../pynocchio/main_window_view.py" line="333"/>
+        <source>Continue reading from page %d?</source>
+        <translation type="unfinished">Kontynuować czytanie ze strony %d?</translation>
+    </message>
+    <message>
+        <location filename="../pynocchio/main_window_view.py" line="336"/>
+        <source>&lt;p&gt;You stopped reading here.&lt;/p&gt;&lt;p&gt; If you choose &lt;b&gt;&quot;Yes&quot;&lt;/b&gt;, reading will resume on &lt;b&gt;page %d&lt;/b&gt;. &lt;/p&gt;&lt;p&gt;Otherwise, the first page will be loaded.&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;p&gt;You stopped reading here.&lt;/p&gt;&lt;p&gt; If you choose &lt;b&gt;&quot;Yes&quot;&lt;/b&gt;, reading will resume on &lt;b&gt;page %d&lt;/b&gt;. &lt;/p&gt;&lt;p&gt;Otherwise, the first page will be loaded.&lt;/p&gt;</source></translation>
```
Comment (kRkk) [inline i18n/pl-pl.ts]: I don't know how translate this, sorry

Comment (mstuttgart) [reply i18n/pl-pl.ts]: No problem. This is a dialog message. Ask to if the user want resume read of the last page that was read or restart on first comic page

**Comment by kRkk** [comment]
So… merge?

**Comment by mstuttgart** [comment]
@kRkk thanks! :+1:

**Pull request merged**